camply/providers/going_to_camp/going_to_camp_provider.py

```diff
@@ -35,44 +35,69 @@
     ),
     "washington.goingtocamp.com": RecreationArea(
         recreation_area="Washington State Parks",
-        recreation_area_id=4,
+        recreation_area_id=3,
         recreation_area_location="Washington, USA",
     ),
     "yukon.goingtocamp.com": RecreationArea(
         recreation_area="Yukon (Backcountry)",
-        recreation_area_id=5,
+        recreation_area_id=4,
         recreation_area_location="Yukon, CA",
     ),
     "hamilton.goingtocamp.com": RecreationArea(
         recreation_area="Hamilton",
-        recreation_area_id=6,
+        recreation_area_id=5,
         recreation_area_location="Ontario, CA",
     ),
     "maitlandvalley.goingtocamp.com": RecreationArea(
         recreation_area="Maitland Valley",
-        recreation_area_id=7,
+        recreation_area_id=6,
         recreation_area_location="Ontario, CA",
     ),
     "orovillepark.goingtocamp.com": RecreationArea(
         recreation_area="Oroville Park",
-        recreation_area_id=8,
+        recreation_area_id=7,
         recreation_area_location="Washington, USA",
     ),
-    "saugreen.goingtocamp.com": RecreationArea(
-        recreation_area="Saugreen Valley",
-        recreation_area_id=9,
+    "saugeen.goingtocamp.com": RecreationArea(
+        recreation_area="Saugeen Valley",
+        recreation_area_id=8,
         recreation_area_location="Ontario, CA",
     ),
     "tacomapower.goingtocamp.com": RecreationArea(
         recreation_area="Tacoma Power Parks",
-        recreation_area_id=10,
+        recreation_area_id=9,
         recreation_area_location="Washington, USA",
     ),
     "wisconsin.goingtocamp.com": RecreationArea(
         recreation_area="Wisconsin State Parks",
-        recreation_area_id=11,
+        recreation_area_id=10,
         recreation_area_location="Wisconsin, USA",
     ),
+    "ahtrails.ca": RecreationArea(
+        recreation_area="Algonquin Highlands",
+        recreation_area_id=11,
+        recreation_area_location="Ontario, CA",
+    ),
+    "parkreservations.maryland.gov": RecreationArea(
+        recreation_area="Maryland State Parks",
+        recreation_area_id=12,
+        recreation_area_location="Maryland, USA",
+    ),
+    "reservations.ncc-ccn.gc.ca": RecreationArea(
+        recreation_area="Gatineau Park",
+        recreation_area_id=13,
+        recreation_area_location="Ottawa-Gatineau, Ontario-Quebec, CA",
+    ),
+    "www.nlcamping.ca": RecreationArea(
+        recreation_area="Newfoundland & Labrador Provincial Parks",
+        recreation_area_id=14,
+        recreation_area_location="Newfoundland and Labrador, CA",
+    ),
+    "camping.bcparks.ca": RecreationArea(
+        recreation_area="BC Parks",
+        recreation_area_id=15,
+        recreation_area_location="British Columbia, CA",
+    ),
```

Comment:
Starting this in a Thread - for some reason GitHub makes it hard to have PR conversations on a thread.

I'm not worried about changing the IDs of GoingToCamp right now - we're out of season and it's a new provider. 

Previously when generating the documentation I just ran the command `camply recreation-areas --provider GoingToCamp` to get the RecArea - and then just pasted the response into the docs. That's why they're in a funky order (which is technically alphabetical).

For Fuzzy matching, I'm a fan of [RapidFuzz](https://github.com/maxbachmann/RapidFuzz) - although it's written in C++ and so requires Python Wheels to simply install (I think RasberryPi wheels might be hard to come by though - and I imagine that might be a common piece of hardware to run camply).

Personally - I think let's just go with numeric IDs - camply users should be familiar with searching for their recreation areas by name in order to retrieve the numeric ID: `camply recreation-areas --provider GoingToCamp --search Tacoma`

Maybe a good numbering system is a prefix for State / Region: i.e. every campground gets a 4 digit code - California is the 5th State in Alphabetical order so it gets campground IDs like 5001 🤷